Source/com/drew/metadata/exif/ExifDescriptorBase.java

```diff
@@ -55,11 +58,17 @@
     // Ev=BV+Sv   Sv=log2(ISOSpeedRating/3.125)
     // ISO100:Sv=5, ISO200:Sv=6, ISO400:Sv=7, ISO125:Sv=5.32.
 
+    // TODO can be removed once context has been added to all sub-classes
```

Comment:
There are 4 sub-classes of this, that are still on the to-do list, so I expect this constructor to be removed in this PR, eventually.